test/moonwall.config.json

```diff
@@ -140,6 +140,72 @@
           "endpoints": ["ws://127.0.0.1:9944"]
         }
       ]
+    },
+    {
+      "name": "update_moonbeam",
```

Comment:
Is there an easy way to run this locally ? As I don't think having the build/xxx.wasm file is standard, does moonwall allow to retrieve/copy it ?

Reply:
the naming scheme you have above is standard for when running through CI, when using the `download_artifact` github action.

For local runs you can always change moonwall config to `target/release/wbuild/moonbeam-runtime/moonbeam_runtime.compact.wasm` if you wished or `moonbeam-runtime-2201.wasm` as downloaded by moonwall.

Reply:
The CI does use the local generated artifacts like you wanted . I should clarify that when I mention download-arrifact, this is the way you share files between jobs (like how typescript tests use the binary from the build step).

Reply:
Done